### PR TITLE
fixed for scalable gunicorns

### PR DESCRIPTION
**Pull request by mira-miracoli**
Fixed non-Ansible-Galaxy roles:
 - usegalaxy-eu.bashrc
 - usegalaxy-eu.fix-stuck-handlers

### CODE REVIEW

File: roles/usegalaxy-eu.fix-stuck-handlers/defaults/main.yml [hunk 1]
```diff
@@ -13,11 +13,10 @@ cron_tasks:
   - name: "Restart gunicorn"
     minute: 30
     hour: 3
-    job: "systemctl restart galaxy-gunicorn"
+    job: "/bin/bash -c 'for (( c=0; c<{{ galaxy_systemd_gunicorns }}; c++ )); do systemctl restart galaxy-gunicorn@$c && sleep {{ sleep_duration }}; done'"
```
Comment (sanjaysrikakulam) [inline roles/usegalaxy-eu.fix-stuck-handlers/defaults/main.yml:16]: Are 4 minutes of sleep duration enough?

Comment (mira-miracoli) [reply roles/usegalaxy-eu.fix-stuck-handlers/defaults/main.yml:16]: I can add a minute or two

Comment (sanjaysrikakulam) [reply roles/usegalaxy-eu.fix-stuck-handlers/defaults/main.yml:16]: Yeah, that would be better I think. Better safe than oops!

**Review by sanjaysrikakulam — APPROVED** [review]

File: roles/usegalaxy-eu.fix-stuck-handlers/defaults/main.yml (hunk 1 above)
Comment (bgruening) [reply roles/usegalaxy-eu.fix-stuck-handlers/defaults/main.yml:16]: Shouldn't we (not now, but later) scan the first gunicorn if its up ... scanning for the port and or socket that exists? Or something in the logs that indicate its up? I think Nate has something in gravity for that?

**Pull request merged**